compilation from source fails on Catalina

I'm seeing:

```
> install.packages("fs", type = "source")
Installing package into ‘/Users/kevinushey/Library/R/3.6/library’
(as ‘lib’ is unspecified)
--- Please select a CRAN mirror for use in this session ---
trying URL 'https://cloud.r-project.org/src/contrib/fs_1.3.1.tar.gz'
Content type 'application/x-gzip' length 812680 bytes (793 KB)
==================================================
downloaded 793 KB

* installing *source* package ‘fs’ ...
** package ‘fs’ successfully unpacked and MD5 sums checked
** using staged installation
** libs
clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I./libuv/include -I. -pthread -I"/Users/kevinushey/Library/R/3.6/library/Rcpp/include" -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk -I/usr/local/include  -fPIC  -g -O2 -pipe -Wall -pedantic -c dir.cc -o dir.o
clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I./libuv/include -I. -pthread -I"/Users/kevinushey/Library/R/3.6/library/Rcpp/include" -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk -I/usr/local/include  -fPIC  -g -O2 -pipe -Wall -pedantic -c error.cc -o error.o
clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I./libuv/include -I. -pthread -I"/Users/kevinushey/Library/R/3.6/library/Rcpp/include" -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk -I/usr/local/include  -fPIC  -g -O2 -pipe -Wall -pedantic -c file.cc -o file.o
clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I./libuv/include -I. -pthread -I"/Users/kevinushey/Library/R/3.6/library/Rcpp/include" -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk -I/usr/local/include  -fPIC  -g -O2 -pipe -Wall -pedantic -c fs.cc -o fs.o
clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I./libuv/include -I. -pthread -I"/Users/kevinushey/Library/R/3.6/library/Rcpp/include" -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk -I/usr/local/include  -fPIC  -g -O2 -pipe -Wall -pedantic -c id.cc -o id.o
clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I./libuv/include -I. -pthread -I"/Users/kevinushey/Library/R/3.6/library/Rcpp/include" -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk -I/usr/local/include  -fPIC  -g -O2 -pipe -Wall -pedantic -c link.cc -o link.o
clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I./libuv/include -I. -pthread -I"/Users/kevinushey/Library/R/3.6/library/Rcpp/include" -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk -I/usr/local/include  -fPIC  -g -O2 -pipe -Wall -pedantic -c path.cc -o path.o
clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I./libuv/include -I. -pthread -I"/Users/kevinushey/Library/R/3.6/library/Rcpp/include" -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk -I/usr/local/include  -fPIC  -g -O2 -pipe -Wall -pedantic -c utils.cc -o utils.o
clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I./libuv/include -I. -pthread -I"/Users/kevinushey/Library/R/3.6/library/Rcpp/include" -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk -I/usr/local/include  -fPIC  -g -O2 -pipe -Wall -pedantic -c unix/getmode.cc -o unix/getmode.o
clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I./libuv/include -I. -pthread -I"/Users/kevinushey/Library/R/3.6/library/Rcpp/include" -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk -I/usr/local/include  -fPIC  -g -O2 -pipe -Wall -pedantic -c RcppExports.cpp -o RcppExports.o
cp -p -f libuv/m4/lt_obsolete.m4 libuv/m4/lt~obsolete.m4
(cd libuv \
                && touch aclocal.m4 \
                && touch -r aclocal.m4 configure Makefile.in \
        && CC="clang" CFLAGS="-Wall -g -O2  -fPIC " AR="ar" RANLIB="ranlib" LDFLAGS="-L/usr/local/lib" ./configure --quiet)
configure: error: in `/private/var/folders/b4/2422hswx71z8mgwtv4rhxchr0000gn/T/RtmpcLR530/R.INSTALLd50063164df2/fs/src/libuv':
configure: error: cannot run C compiled programs.
If you meant to cross compile, use `--host'.
See `config.log' for more details
make: *** [libuv/Makefile] Error 1
make: *** Waiting for unfinished jobs....
ERROR: compilation failed for package ‘fs’
* removing ‘/Users/kevinushey/Library/R/3.6/library/fs’
* restoring previous ‘/Users/kevinushey/Library/R/3.6/library/fs’
```

My suspicion is that the lack of headers in `/usr/include` on Catalina is causing an issue during compilation of libuv.

--- a/NEWS.md
+++ b/NEWS.md
@@ -1,5 +1,8 @@
 # fs (development version)
 
+* fs now passes along CPPFLAGS during compilation of libuv, fixing an issue that could
+  prevent compilation from source on macOS Catalina. (@kevinushey, #229)
+  
 * `dir_tree()` now works with paths that need tilde expansion (@dmurdoch, @jennybc, #203).
 * `is_dir()`, `is_file()`, `is_file_empty()` and `file_info()` gain a `follow`
   argument, to follow links and return information about the linked file rather

--- a/src/Makevars
+++ b/src/Makevars
@@ -32,7 +32,7 @@ libuv/Makefile: libuv/m4/lt~obsolete.m4
 	(cd libuv \
 		&& touch aclocal.m4 \
 		&& touch -r aclocal.m4 configure Makefile.in \
-	&& CC="$(CC)" CFLAGS="$(CFLAGS) $(CPICFLAGS) $(C_VISIBILITY)" AR="$(AR)" RANLIB="$(RANLIB)" LDFLAGS="$(LDFLAGS)" ./configure --quiet)
+	&& CC="$(CC)" CFLAGS="$(CFLAGS) $(CPPFLAGS) $(CPICFLAGS) $(C_VISIBILITY)" AR="$(AR)" RANLIB="$(RANLIB)" LDFLAGS="$(LDFLAGS)" ./configure --quiet)
 
 libuv/.libs/libuv.a: libuv/Makefile
 	$(MAKE) --directory=libuv \

